### DISCUSSION TITLE
Use for Open Water/Lake Evaporation

### DISCUSSION THREAD

**Discussion by dvdjng** [pyet-org/pyet#44] [Q&A] [answered]
Hello everyone

Thanks for the efforts put into this library!
Question: Which evapotranspiration function yields the most accurate results for open water/lake evaporation?
Would be cool to modify PM to allow application for open water - or is something like this already done?

Kind regards
David

**Comment by mvremec** [comment, marked as answer]
Hi David!
Thank you for bringing this up! 
In fact, we are still considering which method to implement for deep and shallow lake evaporation.

I recommend you take a look at the article by McMahon et al. (2013), which presents different methods for estimating evaporation from lakes. It was published in HESS and can be downloaded [here](https://hess.copernicus.org/articles/17/1331/2013/hess-17-1331-2013.html).

According to McMahon et al. (2013), one option is to use the Penman equation to estimate open-water evaporation. 
This can already be done with PyEt. 
I have added one of the working examples from McMahon et al. (2013). You can check it [here](https://github.com/pyet-org/pyet/blob/dev/examples/06_worked_examples_McMahon_etal_2013.ipynb). 
In the developer branch, I also edited the wind function so it better complies with McMahon et al. (2013).

In short, when using the Penman function, use albedo=0.08 for calculating the net radiation.

The next step after calculating Penman open-water evaporation would be to use the procedure of Kohler and Parmele (1967), but this would require data on the temperature of the lake.
If you think the procedure of Kohler and Parmele (1967) would be applicable to your study and would also help others, we could think about implementing it in PyEt.

I hope this helped!

Best,
Matevz

McMahon, T. A., Peel, M. C., Lowe, L., Srikanthan, R., and McVicar, T. R.: Estimating actual, potential, reference crop and pan evaporation using standard meteorological data: a pragmatic synthesis, Hydrol. Earth Syst. Sci., 17, 1331–1363, https://doi.org/10.5194/hess-17-1331-2013, 2013.
Kohler, M. A. and Parmele, L. H.: Generalized estimates of freewater evaporation, Water Resour. Res., 3, 997–1005, 1967.
